const publications = [
  {
    title: "Development of diagnostic markers for the disease susceptibility gene <i>Tsn1</i> in wheat reveals novel resistance alleles and a new locus required for ToxA sensitivity",
    authors: "Running K, Acharya K, Roth T, <strong>Singh G</strong>, Szabo-Hever A, Haugrud A, Fiedler J, Friesen TL, and Faris JD",
    journal: "<i>Theoretical and Applied Genetics</i>",
    year: 2025,
    citations: 0,
    url: "",
    abstract: "This publication is under review.",
    citingPapers: []
  },  
  {
    title: "Evaluation of Field Pea Cultivars for Resistance to Pin Nematode (<i>Paratylenchus nanus</i> type B) in North Dakota",
    authors: "Ojha E, <strong>Singh G</strong>, Plaisance A, and Yan GP",
    journal: "<i>Plant Disease</i>",
    year: 2025,
    citations: 0,
    url: "",
    abstract: "This publication is under review.",
    citingPapers: []
  },  
  {
    title: "Protein kinase-major sperm protein (PK-MSP) genes mediate recognition of the fungal necrotrophic effector SnTox3 to cause septoria nodorum blotch in wheat",
    authors: "Zhang Z, Running K, Seneviratne S, Haugrud A, Szabo-Hever A, <strong>Singh G</strong>, Holušová K, Molnár I, Doležel J, Friesen TL, and Faris JD",
    journal: "<i>Molecular Plant-Microbe Interactions</i>",
    year: 2025,
    citations: 0,
    url: "",
    abstract: "This publication is submitted <i>Molecular Plant-Microbe Interactions</i> and accepted with minor revisions.",
    citingPapers: []
  },  
  {
    title: "Evaluation of durum and hard red spring wheat panels for sensitivity to necrotrophic effectors produced by Parastagonospora nodorum",
    authors: "Szabo-Hever A, Running K, Seneviratne S, <strong>Singh G</strong>, Zhang Z, Haugrud A, Maccaferri M, Tuberosa R, Friesen TL, Xu SS, and Faris JD",
    journal: "<i>Plant Disease</i>",
    year: 2024,
    citations: 0,
    url: "https://doi.org/10.1094/PDIS-05-24-0990-RE",
    abstract: "<strong>Abstract:</strong> Septoria nodorum blotch is an important disease of both durum and hard red spring wheat (HRSW) worldwide. The disease is caused by the necrotrophic fungal pathogen Parastagonospora nodorum when compatible gene-for-gene interactions occur between pathogen-produced necrotrophic effectors (NEs) and corresponding host sensitivity genes. To date, nine sensitivity gene-NE interactions have been identified, but there is little information available regarding their overall frequency in durum and HRSW. Here, we infiltrated a global HRSW panel (HRSWP) and the Global Durum Panel (GDP) with P. nodorum NEs SnToxA, SnTox1, SnTox267, SnTox3, and SnTox5. Frequencies of sensitivity to SnTox1 and SnTox5 were higher in durum compared to HRSW and vice versa for SnTox267 and SnTox3. Strong associations for the known sensitivity loci Tsn1, Snn1, Snn2, Snn3, Snn5, and Snn7 along with potentially novel sensitivity loci on chromosome arms 7DS and 3BL associated with SnToxA and SnTox267, respectively, were identified in the HRSWP. In the GDP, Snn1, Snn3, and Snn5 were identified along with novel loci associated with sensitivity to SnTox267 on chromosome arms 2AS, 2AL, and 6AS and with SnTox5 sensitivity on 2BS and 7BL. These results reveal additional NE sensitivity loci beyond those previously described demonstrating a higher level of genetic complexity of the wheat-P. nodorum system than previously thought. Knowledge regarding the prevalence and genomic locations of SNB susceptibility genes in HRSW and durum will prove useful for developing efficient breeding strategies and improving varieties for SNB resistance.",
    citingPapers: []
  },
  {
    title: "Evolution, diversity, and function of the disease susceptibility gene <i>Snn1</i> in wheat",
    authors: "Seneviratne S, Shi G, Szabo-Hever A, Zhang Z, Haugrud A, Running K, <strong>Singh G</strong>, Nandety R, Fiedler J, McClean PE, Xu SS, Friesen TL, and Faris JD",
    journal: "<i>The Plant Journal</i> 119:1720-1736",
    year: 2024,
    citations: 2,
    url: "https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.16879",
    abstract: "<strong>Abstract:</strong> Septoria nodorum blotch (SNB), caused by <i>Parastagonospora nodorum</i>, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 was previously cloned, and it encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to programmed cell death and ultimately the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid, and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1–SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein–protein interaction. A high-throughput marker was developed for the absent allele of Snn1, and it was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.",
    citingPapers: [
      {
        title: "Evaluation of durum and hard red spring wheat panels for sensitivity to necrotrophic effectors produced by <i>Parastagonospora nodorum</i>",
        authors: "Szabo-Hever A, Running K, Seneviratne S, Singh G, Zhang Z, Haugrud A, Maccaferri M, Tuberosa R, Friesen TL, Xu SS, Faris JD",
        journal: "<em>Plant Disease</em>",
        year: 2024,
        url: "https://doi.org/10.1094/PDIS-05-24-0990-RE"
      },
      {
        title: "Association mapping of tan spot and septoria nodorum blotch resistance in cultivated emmer wheat",
        authors: "Lhamo D, Sun Q, Friesen TL, Karmacharya A, Li X, Fiedler J, Faris JD, Xia G, Luo M, Gu Y, Liu Z, Xu SS",
        journal: "<em>Theoretical and Applied Genetics</em>",
        year: 2024,
        url: "https://doi.org/10.1007/s00122-024-04700-2"
      }
    ]
  },
  {
    title: "Evaluation of wheat cultivars and germplasm lines for resistance to <i>Pratylenchus neglectus</i> populations collected in North Dakota",
    authors: "<strong>Singh G</strong>, KC A, Sandhu KS, Friskop AJ, Liu Z, and Yan GP",
    journal: "<i>Plant Disease</i> 117:3817-3824",
    year: 2023,
    citations: 2,
    url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PDIS-03-23-0590-RE",
    abstract: "<strong>Abstract:</strong> Root-lesion nematode (RLN; <i>Pratylenchus neglectus</i>) is a migratory endoparasite and a major soilborne pathogen that affects wheat (<i>Triticum</i> spp.) production worldwide. Genetic resistance is one of the most economical and effective ways to manage P. neglectus in wheat. This study evaluated 37 local cultivars and germplasm lines in seven greenhouse experiments, including 26 hexaploid wheat, six durum wheat, two synthetic hexaploid wheat, one emmer wheat, and two triticale for P. neglectus resistance from 2016 to 2020. North Dakota field soils infested with two RLN populations (350 to 1,125 nematodes per kilogram of soil) were used for resistance screening under controlled greenhouse conditions. The final nematode population density for each cultivar and line was counted under the microscope to categorize the resistance ranking of these entries as resistant, moderately resistant, moderately susceptible, and susceptible. Out of the 37 cultivars and lines, one was classified as resistant (Brennan); 18 were moderately resistant (Divide, Carpio, Prosper, Advance, Alkabo, SY Soren, Barlow, Bolles, Select, Faller, Briggs, WB Mayville, SY Ingmar, W7984, PI 626573, Ben, Grandin, and Villax St. Jose); 11 were moderately susceptible; and seven were susceptible to P. neglectus. The resistant to moderately resistant lines identified in this study could be used in breeding programs after the resistance genes or loci are further elucidated. This research provides valuable information about P. neglectus resistance among wheat and triticale cultivars used in the Upper Midwest region of the United States.",
    citingPapers: [
      {
        title: "Reproduction and molecular detection of plant-parasitic nematodes associated with potato",
        authors: "Goraya M",
        journal: "<i>MS Thesis, North Dakota State University, Fargo, ND, USA</i>",
        year: 2023,
        url: "https://www.proquest.com/openview/7d141f2e1f4a10955286fc9bf10beab4/1?pq-origsite=gscholar&cbl=18750&diss=y"
      },
      {
        title: "Molecular characterization of pin nematodes and evaluation of resistance in field pea cultivars",
        authors: "Ojha E",
        journal: "<i>MS Thesis, North Dakota State University, Fargo, ND, USA</i>",
        year: 2023,
        url: "https://www.proquest.com/openview/d20a446716a376dad835851bf2959cf6/1?pq-origsite=gscholar&cbl=18750&diss=y"
      }
    ]
  },
  {
    title: "Association mapping of resistance to tan spot in the Global Durum Panel",
    authors: "Szabo-Hever A, <strong>Singh G</strong>, Haugrud A, Running K, Seneviratne S, Zhang Z, Shi G, Bassi F, Maccaferri M, Cattivelli L, Tuberosa R, Friesen TL, Liu Z, Xu SS, and Faris JD",
    journal: "<i>Phytopathology</i> 113:1967-1978",
    year: 2023,
    citations: 5,
    url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PHYTO-02-23-0043-R",
    abstract: "<strong>Abstract:</strong> Tan spot, caused by the necrotrophic fungal pathogen <i>Pyrenophora tritici-repentis</i> (<i>Ptr</i>), is an important disease of durum and common wheat worldwide. Compared with common wheat, less is known about the genetics and molecular basis of tan spot resistance in durum wheat. We evaluated 510 durum lines from the Global Durum Wheat Panel (GDP) for sensitivity to the necrotrophic effectors (NEs) Ptr ToxA and Ptr ToxB and for reaction to Ptr isolates representing races 1 to 5. Overall, susceptible durum lines were most prevalent in South Asia, the Middle East, and North Africa. Genome-wide association analysis showed that the resistance locus Tsr7 was significantly associated with tan spot caused by races 2 and 3, but not races 1, 4, or 5. The NE sensitivity genes Tsc1 and Tsc2 were associated with susceptibility to Ptr ToxC- and Ptr ToxB-producing isolates, respectively, but Tsn1 was not associated with tan spot caused by Ptr ToxA-producing isolates, which further validates that the Tsn1−Ptr ToxA interaction does not play a significant role in tan spot development in durum. A unique locus on chromosome arm 2AS was associated with tan spot caused by race 4, a race once considered avirulent. A novel trait characterized by expanding chlorosis leading to increased disease severity caused by the Ptr ToxB-producing race 5 isolate DW5 was identified, and this trait was governed by a locus on chromosome 5B. We recommend that durum breeders select resistance alleles at the Tsr7, Tsc1, Tsc2, and the chromosome 2AS loci to obtain broad resistance to tan spot.",
    citingPapers: [
      {
        title: "Evaluation of durum and hard red spring wheat panels for sensitivity to necrotrophic effectors produced by <i>Parastagonospora nodorum</i>",
        authors: "Szabo-Hever A, Running K, Seneviratne S, Singh G, Zhang Z, Haugrud A, Maccaferri M, Tuberosa R, Friesen TL, Xu SS, Faris JD",
        journal: "<em>Plant Disease</em>",
        year: 2024,
        url: "https://doi.org/10.1094/PDIS-05-24-0990-RE"
      },
      {
        title: "Evolution, diversity, and function of the disease susceptibility gene <i>Snn1</i> in wheat",
        authors: "Seneviratne S, Shi G, Szabo-Hever A, Zhang Z, Haugrud A, Running K, <strong>Singh G</strong>, Nandety R, Fiedler J, McClean PE, Xu SS, Friesen TL, Faris JD",
        journal: "<em>The Plant Journal</em>",
        year: 2024,
        url: "https://doi.org/10.1111/tpj.16879"
      },
      {
        title: "Association mapping of tan spot and septoria nodorum blotch resistance in cultivated emmer wheat",
        authors: "Lhamo D, Sun Q, Friesen TL, Karmacharya A, Li X, Fiedler J, Faris JD, Xia G, Luo M, Gu Y, Liu Z, Xu SS",
        journal: "<em>Theoretical and Applied Genetics</em>",
        year: 2024,
        url: "https://doi.org/10.1007/s00122-024-04700-2"
      },
      {
        title: "Marker assisted backcross introgression of a race non-specific QTL to develop durum wheat lines with improved tan spot resistance",
        authors: "Acharya N",
        journal: "<i>MS Thesis, North Dakota State University, Fargo, ND, USA</i>",
        year: 2024,
        url: "https://www.proquest.com/docview/3106536557?fromopenview=true&pq-origsite=gscholar&sourcetype=Dissertations%20&%20Theses"
      },
      {
        title: "Genome-wide association mapping of resistance to the foliar diseases septoria nodorum blotch and tan spot in a global winter wheat collection",
        authors: "Haugrud A, Shi G, Seneviratne S, Running K, Zhang Z, <strong>Singh G</strong>, Szabo-Hever A, Acharya K, Friesen TL, Liu Z, Faris JD",
        journal: "<em>Molecular Breeding</em>",
        year: 2023,
        url: "https://doi.org/10.1007/s11032-023-01400-5"
      }
    ]
  },
  {
    title: "Genome-wide association mapping of resistance to the foliar diseases septoria nodorum blotch and tan spot in a global winter wheat collection",
    authors: "Haugrud A, Shi G, Seneviratne S, Running K, Zhang Z, <strong>Singh G</strong>, Szabo-Hever A, Acharya K, Friesen TL, Liu Z, and Faris JD",
    journal: "<i>Molecular Breeding</i> 43:54",
    year: 2023,
    citations: 6,
    url: "https://link.springer.com/article/10.1007/s11032-023-01400-5",
    abstract: "<strong>Abstract:</strong> Septoria nodorum blotch (SNB) and tan spot, caused by the necrotrophic fungal pathogens Parastagonospora nodorum and Pyrenophora tritici-repentis, respectively, often occur together as a leaf spotting disease complex on wheat (Triticum aestivum L.). Both pathogens produce necrotrophic effectors (NEs) that contribute to the development of disease. Here, genome-wide association analysis of a diverse panel of 264 winter wheat lines revealed novel loci on chromosomes 5A and 5B associated with sensitivity to the NEs SnTox3 and SnTox5 in addition to the known sensitivity genes for NEs Ptr/SnToxA, SnTox1, SnTox3, and SnTox5. Sensitivity loci for SnTox267 and Ptr ToxB were not detected. Evaluation of the panel with five P. nodorum isolates for SNB development indicated the Snn3-SnTox3 and Tsn1-SnToxA interactions played significant roles in disease development along with additional QTL on chromosomes 2A and 2D, which may correspond to the Snn7-SnTox267 interaction. For tan spot, the Tsc1-Ptr ToxC interaction was associated with disease caused by two isolates, and a novel QTL on chromosome 7D was associated with a third isolate. The Tsn1-ToxA interaction was associated with SNB but not tan spot. Therefore some, but not all, of the previously characterized host gene-NE interactions in these pathosystems play significant roles in disease development in winter wheat. Based on these results, breeders should prioritize the selection of resistance alleles at the Tsc1, Tsn1, Snn3, and Snn7 loci as well as the 2A and 7D QTL to obtain good levels of resistance to SNB and tan spot in winter wheat.",
    citingPapers: [
      {
        title: "Evaluation of durum and hard red spring wheat panels for sensitivity to necrotrophic effectors produced by <i>Parastagonospora nodorum</i>",
        authors: "Szabo-Hever A, Running K, Seneviratne S, Singh G, Zhang Z, Haugrud A, Maccaferri M, Tuberosa R, Friesen TL, Xu SS, Faris JD",
        journal: "<em>Plant Disease</em>",
        year: 2024,
        url: "https://doi.org/10.1094/PDIS-05-24-0990-RE"
      },
      {
        title: "Association mapping of tan spot and septoria nodorum blotch resistance in cultivated emmer wheat",
        authors: "Lhamo D, Sun Q, Friesen TL, Karmacharya A, Li X, Fiedler J, Faris JD, Xia G, Luo M, Gu Y, Liu Z, Xu SS",
        journal: "<em>Theoretical and Applied Genetics</em>",
        year: 2024,
        url: "https://doi.org/10.1007/s00122-024-04700-2"
      },
      {
        title: "Interaction of wheat-Parastagonospora nodorum isolates: Exploring host susceptibility and fungal virulence",
        authors: "Choupannejad R, Sharifnabi B, Collemare J, Massah A, Gholami J, Mehrabi R",
        journal: "<em>Physiological and Molecular Plant Pathology</em>",
        year: 2024,
        url: "https://doi.org/10.1016/j.pmpp.2023.102235"
      },
      {
        title: "Survival of the Exserohilum rostratum causal leaf spot of wheat and dissemination, infection behaviour from flower-seed-seedlings",
        authors: "Korra T, P. T, Chandra R",
        journal: "<em>Cereal Research Communications</em>",
        year: 2024,
        url: "https://doi.org/10.1007/s42976-024-00575-7"
      },
      {
        title: "Host susceptibility and fungal virulence in the wheat-Parastagonospora nodorum interactions",
        authors: "Choupannejad R, Sharifnabi B, Collemare J, Massah A, Gholami J, Mehrabi R",
        journal: "<em>Research Square</em> (Preprint)",
        year: 2023,
        url: "https://doi.org/10.21203/rs.3.rs-3198290/v1"
      },
      {
        title: "Intelligent reprogramming of wheat for enhancement of fungal and nematode disease resistance using advanced molecular techniques",
        authors: "Jabran M, Ali MA, Zahoor A, Muhae-Ud-Din G, Liu T, Chen W, Gao L",
        journal: "<em>Frontiers in Plant Science</em>",
        year: 2023,
        url: "https://doi.org/10.3389/fpls.2023.1132699"
      }
    ]
  },
  {
    title: "Resistance screening and QTL mapping in wheat and triticale against root-lesion nematode",
    authors: "<strong>Singh G</strong>",
    journal: "<i>MS Thesis</i>, North Dakota State University, Fargo, ND, USA",
    year: 2020,
    citations: 5,
    url: "https://www.proquest.com/openview/f06d4867c07d7e3ea21b2e2a9b5b6cb6/1?pq-origsite=gscholar&cbl=18750&diss=y",
    abstract: "<strong>Abstract:</strong> Root-lesion nematode (RLN, Pratylenchus neglectus) invades the roots of wheat and causes yield losses throughout the world. Genetic resistance is the most economical and effective means to manage RLNs. The objective of this study were to identify source of resistance to RLN in a small collection of wheat germplasm and to map quantitative trait loci (QTL) associated with RLN resistance in two; one wheat and one triticale recombinant inbred line (RIL) populations. Out of wheat lines, three were resistant, including hard red spring wheat cultivars Brennan, SY Ingmar, and SY Soren. A number of genomic regions in wheat and rye were identified as QTL for RLN resistance. My research provides a better understanding of the genetic basis of P. neglectus resistance and important tools for RLN resistance breeding.",
    citingPapers: [
      {
        title: "Reproduction and molecular detection of plant-parasitic nematodes associated with potato",
        authors: "Goraya M",
        journal: "<em>MS Thesis, North Dakota State University, Fargo, ND, USA</em>",
        year: 2023,
        url: "https://www.proquest.com/openview/7d141f2e1f4a10955286fc9bf10beab4/1?pq-origsite=gscholar&cbl=18750&diss=y"
      },
      {
        title: "Evaluation of wheat cultivars and germplasm lines for resistance to <i>Pratylenchus neglectus</i> populations collected in North Dakota",
        authors: "Singh G, KC A, Sandhu KS, Friskop AJ, Liu Z, Yan GP",
        journal: "<em>Plant Disease</em>",
        year: 2023,
        url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PDIS-03-23-0590-RE"
      },
      {
        title: "Population Development of the Root-Lesion Nematode <i>Pratylenchus dakotaensis</i> on Soybean Cultivars",
        authors: "Chowdhury IA",
        journal: "<em>Plant Disease</em>",
        year: 2022,
        url: "https://apsjournals.apsnet.org/doi/full/10.1094/PDIS-11-21-2444-RE"
      },
      {
        title: "Wheat Landraces Versus Resistance to Biotic and Abiotic Stresses",
        authors: "Shlibak A, Örgeç M, Zencirci N",
        journal: "<em>In book: Wheat Landraces. (Edition-1) Chapter-10</em>",
        year: 2021,
        url: "https://link.springer.com/chapter/10.1007/978-3-030-77388-5_10"
      },
      {
        title: "Plant-Parasitic Nematodes on Corn (<i>Zea mays</i> L.) and Soybean (<i>Glycine max</i> L.) in North Dakota",
        authors: "Chowdhury IA",
        journal: "<em>PhD Thesis, North Dakota State University, Fargo, ND, USA</em>",
        year: 2020,
        url: "https://www.proquest.com/docview/2479795157?fromopenview=true&pq-origsite=gscholar&sourcetype=Dissertations%20&%20Theses"
      }
    ]
  }
];



const conferences = [
  {
    title: "Genomic and Functional Characterization of an <i>Exo70-like</i> Gene Governing Susceptibility to Tan Spot in Wheat",
    authors: "<strong>Singh G</strong>, Running K, Seneviratne S, Acharya K, Roth T, Haugrud A, Zhang Z, Szabo-Hever A, Kumari P, Liu Z, and Faris JD",
    conference: "<i>Plant and Animal Genome XXXII (PAG 32)</i> Conference, San Diego, California",
    year: 2025,
    citations: 0,
    url: "https://plan.core-apps.com/pag32/abstract/0d2898dc-9da9-49f8-8f92-e5c77817cf59",
    abstract: "Tan spot, caused by the necrotrophic fungus <i>Pyrenophora tritici-repentis</i>, leads to significant yield losses in wheat worldwide. The fungal effector Ptr ToxB interacts with the wheat susceptibility gene <i>Tsc2</i>, inducing chlorosis and reducing photosynthetic activity. This study aimed to identify and functionally characterize <i>Tsc2</i> to enhance understanding of this host-pathogen interaction. We mapped the <i>Tsc2</i> locus using a biparental population derived from a cross between durum wheat cultivars Kronos (Ptr ToxB-sensitive) and Gredho (insensitive). High-resolution mapping was conducted in an F2 population of 1,490 individuals (Kronos × Gredho) using KASP markers designed from exome capture data, narrowing <i>Tsc2</i> to a 108-kb interval on chromosome arm 2BS containing ten candidate genes. Comparative genomics across ten wheat genome assemblies revealed that two genes, an <i>Exo70</i> and an <i>LRR-Protein Kinase</i>, were present exclusively in sensitive genotypes. Sequencing of Kronos TILLING mutants and Ptr ToxB-insensitive EMS mutants of the cultivar Prosper identified mutations in Exo70 correlating with insensitivity, implicating it as the candidate <i>Tsc2</i> gene. Functional validation is underway through genetic complementation using <i>Agrobacterium</i>-mediated transformation of insensitive genotypes (Bobwhite and Fielder) with the wild-type <i>Exo70</i> gene and CRISPR/Cas9-mediated gene knockouts. Additionally, yeast two-hybrid assays are being conducted to elucidate the interaction between Exo70 and Ptr ToxB. The identification and functional characterization of <i>Exo70</i> advances our understanding of the molecular mechanisms underlying wheat susceptibility to tan spot. We developed a high-throughput marker for the absent <i>Tsc2</i> allele, efficiently predicting Ptr ToxB-insensitive lines in durum and spring wheat, facilitating marker-assisted elimination of <i>Tsc2</i>.",
    citingPapers: []
  },
  {
    title: "Characterization of wheat genes governing sensitivity to the <i>Parastagonospora nodorum</i> necrotrophic effector SnTox5",
    authors: "Running K, Sharma S, Seneviratne S, Poddar S, Szabo-Hever A, Haugrud A, Pfaff T, <b>Singh G</b>, Acharya K, Kariyawasam G, Zhang Z, Staskawicz BJ, Xu SS, Friesen TL, and Faris JD",
    conference: "<i>International Symposium on Cereal Leaf Blight</i>, Zurich, Switzerland",
    year: 2024,
    citations: 0,
    url: "",
    abstract: "The necrotrophic fungal pathogen Parastagonospora nodorum produces necrotrophic effectors (NEs) that target specific genes in wheat to cause the disease septoria nodorum blotch (SNB). The interaction between the NE SnTox5 and the wheat gene <i>Snn5</i> plays a significant role in the development of SNB. <i>Snn5</i> was previously mapped to the long arm of chromosome 4B. Here, we combined positional cloning, TILLING, and gene editing approaches to clone <i>Snn5</i>. <i>Snn5</i> encodes a protein with protein kinase and major sperm protein (PK-MSP) domains expressed in multiple tissues but primarily roots. Analysis of <i>Snn5</i>-disrupted mutants indicated that both major domains are essential for function. Through related genetic experiments, we identified a second SnTox5-sensitivity locus on the short arm of chromosome 2B. This gene has been delineated to an 8.54 cM interval corresponding to a candidate gene region spanning 9.6 Mb in Svevo RefSeq v 1.0 containing 232 genes, six of which are <i>PK-MSP-like</i> genes. Given the presence of multiple SnTox5 sensitivity genes, we propose that the original sensitivity gene mapped to chromosome 4B be termed <i>Snn5-B1</i> and the second sensitivity gene mapped to chromosome 2B be termed <i>Snn5-B2</i>. Genome-wide association studies on large panels of durum and common wheat indicated that <i>Snn5-B1</i> was predominant among common wheat accessions whereas <i>Snn5-B2</i> appeared to be relatively more frequent among durum lines. This research extends our knowledge of the Snn5-SnTox5 interaction and the wheat-P. nodorum system in general, and provides tools to aid in the development of SNB-resistant wheat via marker-assisted selection and gene editing.",
    citingPapers: []
  },
  {
    title: "Molecular cloning of the septoria nodorum blotch susceptibility gene <i>Snn2</i> in wheat",
    authors: "Seneviratne S, Zhang Z, Running K, Peters Haugrud AR, Szabo-Hever A, <strong>Singh G</strong>, Acharya K, Fiedler JD, Mclean P, Xu SS, Yang S, Friesen TL, and Faris JD",
    conference: "<i>Plant and Animal Genome XXXI (PAG 31)</i> Conference, San Diego, California",
    year: 2024,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=410372",
    abstract: "Septoria nodorum blotch (SNB) of wheat is a destructive foliar and glume disease caused by the necrotrophic fungal pathogen <i>Parastagonospora nodorum</i>. The disease is initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by dominant host sensitivity genes in wheat in an inverse gene-for-gene manner. Multiple host sensitivity genes and NEs have been identified in this pathosystem, and some NEs interact with more than one host sensitivity gene. <i>Snn2</i>, which is located on the chromosome 2DS, is the first host gene that was found to be interacting with the necrotrophic effector SnTox267. The objective of this study is to clone and characterize the Snn2 gene. Towards the molecular cloning of <i>Snn2</i>, a saturated genetic linkage map was constructed using a segregating population of 164 F7:8 recombinant inbred lines. The map was saturated with markers identified through SNP genotyping using the 90K iSelect wheat SNP chip, as well as simple sequence repeat (SSR) markers developed based on whole-genome sequence scaffolds, wheat survey sequences, and the IWGSC RefSeq v1.0 reference genome. A high-resolution map was then developed using an F2 population derived from a cross between the SnTox267-insensitive wheat line BR34 and the SnTox267-sensitive line BG301. Over 10,000 gametes were screened for high resolution mapping, and <i>Snn2</i> was delineated to a genetic interval of 0.58 cM, which corresponds to a physical segment of 0.96 Mb on Chinese Spring reference genome. A total of 45 high confidence genes were identified in this region, which included eight NB-ARC domain-containing genes and ten protein kinase (PK)-containing genes. Analyses of EMS-induced SnTox267-insensitive mutants and genome-wide association studies indicated that one of the PK-major sperm protein (PK-MSP) genes is a strong candidate for Snn2. CRISPR/Cas9-mediated gene editing revealed that the inactivation of this candidate gene can transform the sensitive wheat variety Bobwhite to become insensitive to SnTox267, confirming it as the Snn2 gene. Currently, a gene complementation experiment is being carried out to further validate the cloning of Snn2. Additionally, work is in progress to characterize <i>Snn2</i> and other SnTox267-sensitivity genes. The findings of this study increase our understanding of wheat-<i>P. nodorum</i> pathosystem and will help to develop better host resistance against SNB through genetic manipulation.",
    citingPapers: []
  },
  {
    title: "Molecular mapping of a novel genetic locus governing enhanced susceptibility to tan spot in durum wheat",
    authors: "Pfaff T, <strong>Singh G</strong>, Running K, Szabo-Hever A, and Faris JD",
    conference: "<i>Plant and Animal Genome XXXI (PAG 31)</i> Conference, San Diego, California",
    year: 2024,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=410373",
    abstract: "Tan spot is an important foliar disease in wheat caused by the fungal pathogen Pyrenophora tritici-repentis (Died.) Drechs. (Ptr). The wheat-Ptr pathosystem follows an inverse gene-for-gene model where pathogen-produced necrotrophic effectors (NEs) are recognized by dominant host genes resulting in host-induced programmed cell death, which leads to disease in the plant. The NE Ptr ToxB is recognized by the gene Tsc2, which results in a compatible interaction that leads to induction of chlorosis in both durum and common wheat. In genome wide association studies (GWAS) on the Global Durum Panel, we observed that infiltration of Ptr ToxB into leaves of some wheat lines showed an expansion of chlorosis outside the infiltrated area to the distal end of the leaf, and this expansion was associated with increased disease severity. The locus responsible for chlorosis expansion, which we have designated as Tsc3, was located on chromosome arm 5BL in durum wheat. To precisely map the Tsc3 locus and initiate map-based cloning of the gene, we developed an F2 population derived from durum varieties Kronos and Simeto. Kronos and Simeto are sensitive to Ptr ToxB, but only Simeto expresses expansion of chlorosis outside the infiltration site. Initial low-resolution mapping was completed using 342 F2 plants and 9 KASP markers developed from the polymorphic SNPs identified in the Infinium iSelect 90K SNP array. For saturation mapping, an additional 15 KASP markers were designed from polymorphic SNPs identified by alignment of genes annotated in the Svevo.v1 reference assembly to the Kronos scaffold assemblies and a Simeto assembly. Using 675 F2 plants and the 24 KASP markers, we have delineated the Tsc3 locus to a 1.1 cM interval and identified nine markers that cosegregate with the gene. The Tsc3 locus resides within a 3.2 Mb physical interval on chromosome arm 5BL in the Svevo.v1 reference genome, which contains about 40 high-confidence genes. High-resolution mapping of the locus using approximately 2,300 F2 plants has been initiated to resolve the markers and further narrow the candidate gene interval. In addition, an ethylmethane-sulfonate (EMS)-induced mutant population is currently being developed to identify putative Tsc3-disrupted mutants for gene validation. This work will shed light on the molecular mechanisms operating in the wheat-Ptr pathosystem and aid in the development of new strategies to develop tan spot resistant wheat. ",
    citingPapers: []
  },
  {
    title: "Molecular Cloning and Validation of Candidate Genes for the Tan Spot Susceptibility Locus <i>Tsc2</i> in Wheat",
    authors: "<strong>Singh G</strong>, Running K, Haugrud A, Seneviratne S, Zhang Z, Szabo-Hever A, Acharya K, Liu Z, and Faris JD",
    conference: "<i>ASA, CSSA, SSSA International Annual Meeting</i>, St. Louis, Missouri",
    year: 2023,
    citations: 0,
    url: "https://scisoc.confex.com/scisoc/2023am/meetingapp.cgi/Paper/148627",
    abstract: "Tan spot, a prevalent foliar disease caused by the necrotrophic fungus Pyrenophora tritici-repentis, causes significant yield losses in wheat worldwide. The Tsc2 locus on chromosome arm 2BS in wheat recognizes the fungal necrotrophic effector Ptr ToxB. The Tsc2-Ptr ToxB interaction triggers chlorosis, which leads to decreased photosynthetic activity, plant stress, and ultimately yield loss. In this study, we used a durum wheat recombinant inbred line population derived from a cross between Kronos (sensitive to Ptr ToxB) and Gredho (insensitive), along with a diverse hard red spring wheat panel, to precisely map the Tsc2 locus. To saturate the Tsc2 genomic region, KASP markers were designed using exome capture data from Kronos and Gredho. High-resolution mapping of these molecular markers delineated the Tsc2 locus to a 0.13 cM interval in an F2 population (Kronos × Gredho) consisting of 2,980 gametes. Based on the genome assembly of the durum wheat cultivar Simeto, the interval was 108.48 kb in size and harbored ten candidate genes. These genes were interrogated using a reverse genetics approach with the Kronos TILLING mutants. We also developed an ethyl methanesulfonate (EMS)-induced mutant population of Prosper, and screening of M2 families led to the identification of multiple Ptr ToxB-insensitive mutants. Comparative sequence analysis of multiple Kronos and Prosper EMS mutants revealed a single candidate gene as Tsc2 and is currently being further validated by genetic complementation. Transcription expression levels, allelic diversity, and functional analysis of the gene are also being conducted. The findings of this study will expedite the functional characterization of the Tsc2 gene, enhance our understanding of the wheat-P. tritici-repentis interaction, and aid in developing diagnostic genetic markers for breeding programs to mitigate tan spot in wheat.",
    citingPapers: []
  },
  {
    title: "Development of diagnostic molecular markers for the multi-pathogen susceptibility gene <i>Tsn1</i>",
    authors: "Running K, Pfaff T, <strong>Singh G</strong>, Szabo-Hever A, Peters Haugrud A, Zhang Z, Seneviratne S, McClean PE, Fiedler JD, Friesen TL, and Faris JD",
    conference: "<i>Plant and Animal Genome XXX (PAG 30)</i> Conference, San Diego, California",
    year: 2023,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=399522",
    abstract: "The Tsn1 gene confers sensitivity to the necrotrophic effector ToxA, which is produced by necrotrophic fungal pathogens that cause the foliar diseases tan spot, septoria nodorum blotch, and spot blotch. Codominant microsatellite markers that delineate Tsn1 to 351 kb were developed previously, but they are not amenable to high-throughput genotyping (HTG), and their efficacy was not evaluated. Here, the published genome assemblies of fifteen bread wheat, two durum wheat, and one wild emmer wheat accessions were used to compare gene and transposable element content in Tsn1- and Tsn1+ lines, revealing two conserved haplotypes. Because Tsn1 is almost always null in insensitive cultivars, conserved KASP markers suitable for HTG were designed in the closest syntenic region between Tsn1- and Tsn1+ assemblies. The flanking KASP markers delineated Tsn1 to 135 kb and were validated on over 1,500 diverse lines. Phenotyping of the lines revealed that the markers correctly predicted a ToxA insensitive phenotype in 99.13-100% of the accessions. The KASP markers were less effective at predicting a ToxA sensitive phenotype (88.68-94.17%) due to a preponderance of insensitive lines with sensitive marker alleles. Sequencing analysis of Tsn1 from these lines revealed some carried nonfunctional alleles, while others carried the dominant Tsn1 allele. Future expression analysis and genetic mapping will elucidate the nature of the ToxA insensitive status in lines with the Tsn1 allele. The markers presented here could be used for reliable and robust marker-assisted elimination of Tsn1, furthering the development of wheat genetically resistant to multiple pathogens. ",
    citingPapers: []
  },
  {
    title: "Towards the molecular cloning of tan spot susceptibility gene <i>Tsc2</i> in wheat",
    authors: "<strong>Singh G</strong>, Running K, Peters Haugrud A, Seneviratne S, Zhang Z, Szabo-Hever A, Acharya K, Liu Z, Dubcovsky J, and Faris JD",
    conference: "<i>Plant and Animal Genome XXX (PAG 30)</i> Conference, San Diego, California",
    year: 2023,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=399524",
    abstract: "Tan spot, caused by the necrotrophic fungal pathogen Pyrenophora tritici-repentis, is an important foliar disease that causes significant yield losses in wheat worldwide. The Tsc2 locus on chromosome arm 2BS in wheat recognizes the necrotrophic effector Ptr ToxB to induce a compatible response leading to chlorosis and loss of photosynthetic area on infected wheat leaves, which leads to plant stress and yield loss. In this study, a durum wheat bi-parental population derived from a cross between Kronos (sensitive to Ptr ToxB) and Gredho (insensitive) and a large hard red spring wheat panel were used to delimited the Tsc2 locus to a genetic distance of 1.41 cM, which corresponded to a physical region of 1.37 Mb. The candidate gene region contained 27 putative genes based on the durum wheat reference sequence from Svevo (insensitive to Ptr ToxB). KASP markers were designed to saturate the Tsc2 region using exome capture data from Kronos and Gredho. The molecular markers developed in this study are currently being used for high-resolution mapping of the Tsc2 locus in a large F2 (Kronos × Gredho) population, and candidate genes are being evaluated in several mutant populations. Results from this study will help us clone the Tsc2 gene, gain knowledge and insights regarding the wheat-P. tritici-repentis pathosystem, and develop diagnostic genetic markers to be used in breeding programs to combat tan spot in wheat.",
    citingPapers: []
  },
  {
    title: "Genome-wide association mapping of resistance to the foliar diseases septoria nodorum blotch and tan spot in a global winter wheat (<i>Triticum aestivum</i> L.) collection",
    authors: "Peters Haugrud A, Shi G, Seneviratne S, Running K, Zhang Z, <strong>Singh G</strong>, Szabo-Hever A, Acharya K, Friesen TL, Liu Z, and Faris JD",
    conference: "<i>Plant and Animal Genome XXX (PAG 30)</i> Conference, San Diego, California",
    year: 2023,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=399529",
    abstract: "Septoria nodorum blotch (SNB) and tan spot, caused by the necrotrophic fungal pathogens Parastagonospora nodorum and Pyrenophora tritici-repentis, respectively, are major foliar diseases on wheat (Triticum aestivum L.). Both pathogens produce necrotrophic effectors (NEs) that are known to play significant roles in the development of disease. We evaluated a panel of 264 winter wheat lines representing a large proportion of the global genetic diversity. We found that 38.6, 4.3, 36.7, 39.0, 48.9, and 42.4 % of the lines were sensitive to the NEs Sn/Ptr ToxA, Ptr ToxB, SnTox1, SnTox267, SnTox3, and SnTox5, respectively. We inoculated the panel with fungal spores of five P. nodorum isolates and four P. tritici-repentis isolates and performed genome-wide association mapping to identify genomic regions associated with disease. For SNB, the Snn3-SnTox3 and Tsn1-SnToxA interactions played significant roles in disease development, along with an uncharacterized QTL on chromosome 2A. For tan spot, the Tsc1-Ptr ToxC interaction was associated with disease caused by isolates Pti2 and AR CrossB10, and a novel QTL was identified on chromosome 7D. The Tsn1-ToxA interaction played an important role in SNB development but not in tan spot development. The screening results with purified NEs will be useful for cloning, validation, and marker development of the sensitivity genes Snn3-B1, Snn2, Snn5, Snn6, and Snn7. This work shows the complexity of disease resistance to SNB and tan spot and illustrates that not all of the previously characterized interactions in these pathosystems play a significant role in disease development in winter wheat.",
    citingPapers: []
  },
  {
    title: "Genetic diversity and marker-assisted elimination of <i>Snn1</i> in wheat",
    authors: "Seneviratne S, Shi G, Szabo-Hever A, Zhang Z, Peters Haugrud A, Running K, <strong>Singh G</strong>, Nandety RS, Fiedler JD, Xu SS, Friesen TL, McClean PE, and Faris JD",
    conference: "<i>Plant and Animal Genome XXX (PAG 30)</i> Conference, San Diego, California",
    year: 2023,
    citations: 0,
    url: "https://www.ars.usda.gov/research/publications/publication/?seqNo115=399503",
    abstract: "Septoria nodorum blotch (SNB) is a severe foliar and glume disease of durum and common wheat characterized by cell death that is initiated by interactions between necrotrophic effectors (NEs) produced by the fungus Parastagonospora nodorum and corresponding host sensitivity genes in wheat. The Snn1-SnTox1 interaction is one of multiple host sensitivity gene-NE interactions identified in the wheat-P. nodorum pathosystem, and it plays a significant role in SNB development. The host sensitivity gene Snn1 was previously cloned from wheat, but a diagnostic PCR marker has not been developed. In this study, the allelic diversity of Snn1 was studied to identify causal polymorphisms and to develop markers useful for marker-assisted elimination (MAE) of functional Snn1 alleles. The full length Snn1 gene was sequenced from 99 accessions including diploid, tetraploid, and hexaploid wheat species. Multiple sequence alignments revealed 27 haplotypes based on the coding sequence of the gene. Three mutations that possibly changed the primitive insensitive allele in wild emmer into the sensitive allele in domesticated wheat and four independent SNPs that subsequently changed the sensitive allele into insensitive alleles were identified. These SNPs are within each of the three exons of Snn1 affecting multiple domains. Kompetitive allele specific PCR (KASP) markers were developed for three of these mutations and the Snn1 null allele. These newly developed markers identify more than 95% of the durum and spring wheat accessions that are sensitive to SnTox1 and can be used for MAE. Results of this study also increase our understanding of the evolution of Snn1.",
    citingPapers: []
  },
  {
    title: "",
    authors: "<strong>Singh G</strong>, Yan GP, Liu Z, and Faris JD",
    conference: "<i>7th International Congress of Nematology</i>, Antibes Juan-les-Pins, France",
    year: 2022,
    citations: 0,
    url: "https://www.esn-online.org/assets/Uploads/meetings/e9d266eb07/Book-of-Abstracts_ICN2022.pdf",
    abstract: "The root-lesion nematode (RLN), Pratylenchus neglectus, is an important pathogen of wheat. Utilizing host resistance is the most economical and environment-friendly way to control this nematode. To date, no RLN- resistant commercial wheat cultivar has been reported, and sources of resistance in wheat are scarce. Previous studies have reported high levels of resistance in rye and triticale. Triticale is a hybrid of wheat and rye, and identifying RLN resistance genes in triticale would be the first step to utilizing triticale resistance in wheat. In this study, a triticale population consisting of 141 recombinant inbred lines (RILs) derived from a cross between Siskiyou (susceptible parent) and Villax St. Jose (moderately resistant parent) was evaluated for reaction to P. neglectus under greenhouse conditions. Our goal was to identify QTL (quantitative trait loci) associated with resistance to RLN. The population was genotyped using genotyping-by-sequencing (GBS) and SSR markers, and a high-density genetic linkage map was constructed covering all wheat and rye chromosomes. Single trait multiple-interval mapping analysis revealed the presence of a single QTL (LOD = 8.6) on rye chromosome 5R that accounted for approximately 24.6% of the total phenotypic variation. The identified QTL needs to be validated in different genetic backgrounds to confirm its stability. The PCR-based molecular markers associated with this QTL will be useful for fine-mapping and to better understand the genetics of RLN resistance. This work provides a foundation for the introgression of rye-derived P. neglectus resistance into wheat germplasm.",
    citingPapers: []
  },
  {
    title: "Identification of QTL associated with resistance to root-lesion nematode (<i>Pratylenchus neglectus</i>) in wheat using genotyping-by-sequencing",
    authors: "<strong>Singh G</strong>, Li X, Smiley R, and Yan GP",
    conference: "<i>APS Plant Health 2019 Annual Meeting</i>, Cleveland, Ohio",
    year: 2019,
    citations: 0,
    url: "https://doi.org/10.1094/PHYTO-109-10-S2.1",
    abstract: "The root-lesion nematode, <i>Pratylenchus neglectus</i> is one of the major soil-borne pathogens that reduces wheat productivity throughout the world. As very few management practices are available, utilization of host resistance is considered to be the most effective and preferred method to mitigate effects of root-lesion nematode in wheat. Phenotyping of resistance to <i>P. neglectus</i> in wheat is labor- intensive and time-consuming, which is the major constraint in breeding programs to develop resistant cultivars. Molecular marker assisted selection may enhance the process of developing resistant cultivars. An F2:5 recombinant inbred line population was developed from a cross between a soft white spring wheat “Louise” (susceptible parent) and an Iranian landrace “Persia 20” (resistant parent). The recombinant inbred lines were evaluated for <i>P. neglectus</i> resistance reactions under controlled greenhouse conditions, and then were genotyped using genotyping-by-sequencing (GBS). Using Tassel5 GBSv2 Pipeline, a total of 2,427 high-quality SNPs were identified and used to construct a linkage map. We detected one major QTL (LOD = 5.3) showing association with <i>P. neglectus</i> resistance, located on chromosome 4A and accounted for 35.4% of the total phenotypic variation. The SNP markers closely linked with this QTL will be converted to PCR-based markers that may provide an efficient tool to breeders for developing <i>P. neglectus</i>-resistant cultivars using marker-assisted selection.",
    citingPapers: []
  },
  {
    title: "Methodology to study the resistance of root-lesion nematode (<i>Pratylenchus neglectus</i>) in wheat under greenhouse conditions",
    authors: "<strong>Singh G</strong> and Yan GP",
    conference: "<i>APS North Central Divison Meeting</i>, Fargo, North Dakota",
    year: 2018,
    citations: 1,
    url: "https://doi.org/10.1094/PHYTO-108-12-S2.20",
    abstract: "The root-lesion nematode (RLN; <i>Pratylenchus neglectus</i>) is one of the important plant-parasitic nematodes which invade the roots of wheat and cause yield losses throughout the world. Resistance of wheat cultivars against RLN is traditionally evaluated by measuring reproductive factor (RF) which is calculated by dividing the final nematode population density by the initial population density. However, RF is affected by several factors, such as plant growth period in greenhouse, and soil texture. The objective of this study was to evaluate the effects of these factors on RF. In one experiment, field soil naturally infested with <i>P. neglectus</i> was used to infect wheat plants, and nematodes were extracted at 8, 10, 12, 14, and 16 weeks after planting using Whitehead Tray method. In another experiment, the wheat was grown in two different soil types including sandy and sandy loam soil which were artificially inoculated with 300 <i>P. neglectus</i>/150 g of soil with five replications. A maximum plant root mass and nematode RF of 16 were obtained at 14 weeks after planting. Sandy soil, having RF of 8, had significantly greater reproduction than sandy loam soil (RF = 1). Using sandy soil and harvesting at 14 weeks after planting produced the best results. This improved methodology will facilitate resistance screening of wheat germplasm or varieties against RLN under greenhouse conditions.",
    citingPapers: [
      {
        title: "Evaluation of wheat cultivars and germplasm lines for resistance to <i>Pratylenchus neglectus</i> populations collected in North Dakota",
        authors: "<strong>Singh G</strong>, KC A, Sandhu KS, Friskop AJ, Liu Z, and Yan GP",
        journal: "<i>Plant Disease</i> 117:3817-3824",
        year: 2023,
        url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PDIS-03-23-0590-RE"
      },
    ]
  }
];

