const publications = [
  {
    title: "Evaluation of durum and hard red spring wheat panels for sensitivity to necrotrophic effectors produced by Parastagonospora nodorum",
    authors: "Szabo-Hever A, Running K, Seneviratne S, <strong>Singh G</strong>, Zhang Z, Haugrud A, Maccaferri M, Tuberosa R, Friesen TL, Xu SS, and Faris JD",
    journal: "<i>Plant Disease</i> (Under Review)",
    year: 2024,
    citations: 0,
    url: "#",
    abstract: "Abstract for this publication is not available as it is under review.",
    citingPapers: []
  },
  {
    title: "Evolution, diversity, and function of the disease susceptibility gene <i>Snn1</i> in wheat.",
    authors: "Seneviratne S, Shi G, Szabo-Hever A, Zhang Z, Haugrud A, Running K, <strong>Singh G</strong>, Nandety R, Fiedler J, McClean PE, Xu SS, Friesen TL, and Faris JD",
    journal: "<i>The Plant Journal</i> 119:1720-1736",
    year: 2024,
    citations: 1,
    url: "https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.16879",
    abstract: "<strong>Abstract:</strong> Septoria nodorum blotch (SNB), caused by Parastagonospora nodorum, is a disease of durum and common wheat initiated by the recognition of pathogen-produced necrotrophic effectors (NEs) by specific wheat genes. The wheat gene Snn1 was previously cloned, and it encodes a wall-associated kinase that directly interacts with the NE SnTox1 leading to programmed cell death and ultimately the development of SNB. Here, sequence analysis of Snn1 from 114 accessions including diploid, tetraploid, and hexaploid wheat species revealed that some wheat lines possess two copies of Snn1 (designated Snn1-B1 and Snn1-B2) approximately 120 kb apart. Snn1-B2 evolved relatively recently as a paralog of Snn1-B1, and both genes have undergone diversifying selection. Three point mutations associated with the formation of the first SnTox1-sensitive Snn1-B1 allele from a primitive wild wheat were identified. Four subsequent and independent SNPs, three in Snn1-B1 and one in Snn1-B2, converted the sensitive alleles to insensitive forms. Protein modeling indicated these four mutations could abolish Snn1–SnTox1 compatibility either through destabilization of the Snn1 protein or direct disruption of the protein–protein interaction. A high-throughput marker was developed for the absent allele of Snn1, and it was 100% accurate at predicting SnTox1-insensitive lines in both durum and spring wheat. Results of this study increase our understanding of the evolution, diversity, and function of Snn1-B1 and Snn1-B2 genes and will be useful for marker-assisted elimination of these genes for better host resistance.",
    citingPapers: []
  },
  {
    title: "Evaluation of wheat cultivars and germplasm lines for resistance to <i>Pratylenchus neglectus</i> populations collected in North Dakota.",
    authors: "<strong>Singh G</strong>, KC A, Sandhu KS, Friskop AJ, Liu Z, and Yan GP",
    journal: "<i>Plant Disease</i> 117:3817-3824",
    year: 2023,
    citations: 2,
    url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PDIS-03-23-0590-RE",
    abstract: "<strong>Abstract:</strong> Root-lesion nematode (RLN; Pratylenchus neglectus) is a migratory endoparasite and a major soilborne pathogen that affects wheat (<i>Triticum</i> spp.) production worldwide. Genetic resistance is one of the most economical and effective ways to manage P. neglectus in wheat. This study evaluated 37 local cultivars and germplasm lines in seven greenhouse experiments, including 26 hexaploid wheat, six durum wheat, two synthetic hexaploid wheat, one emmer wheat, and two triticale for P. neglectus resistance from 2016 to 2020. North Dakota field soils infested with two RLN populations (350 to 1,125 nematodes per kilogram of soil) were used for resistance screening under controlled greenhouse conditions. The final nematode population density for each cultivar and line was counted under the microscope to categorize the resistance ranking of these entries as resistant, moderately resistant, moderately susceptible, and susceptible. Out of the 37 cultivars and lines, one was classified as resistant (Brennan); 18 were moderately resistant (Divide, Carpio, Prosper, Advance, Alkabo, SY Soren, Barlow, Bolles, Select, Faller, Briggs, WB Mayville, SY Ingmar, W7984, PI 626573, Ben, Grandin, and Villax St. Jose); 11 were moderately susceptible; and seven were susceptible to P. neglectus. The resistant to moderately resistant lines identified in this study could be used in breeding programs after the resistance genes or loci are further elucidated. This research provides valuable information about P. neglectus resistance among wheat and triticale cultivars used in the Upper Midwest region of the United States.",
    citingPapers: []
  },
  {
    title: "Association mapping of resistance to tan spot in the Global Durum Panel.",
    authors: "Szabo-Hever A, <strong>Singh G</strong>, Haugrud A, Running K, Seneviratne S, Zhang Z, Shi G, Bassi F, Maccaferri M, Cattivelli L, Tuberosa R, Friesen TL, Liu Z, Xu SS, and Faris JD",
    journal: "<i>Phytopathology</i> 113:1967-1978",
    year: 2023,
    citations: 3,
    url: "https://apsjournals.apsnet.org/doi/abs/10.1094/PHYTO-02-23-0043-R",
    abstract: "<strong>Abstract:</strong> Tan spot, caused by the necrotrophic fungal pathogen Pyrenophora tritici-repentis (Ptr), is an important disease of durum and common wheat worldwide. Compared with common wheat, less is known about the genetics and molecular basis of tan spot resistance in durum wheat. We evaluated 510 durum lines from the Global Durum Wheat Panel (GDP) for sensitivity to the necrotrophic effectors (NEs) Ptr ToxA and Ptr ToxB and for reaction to Ptr isolates representing races 1 to 5. Overall, susceptible durum lines were most prevalent in South Asia, the Middle East, and North Africa. Genome-wide association analysis showed that the resistance locus Tsr7 was significantly associated with tan spot caused by races 2 and 3, but not races 1, 4, or 5. The NE sensitivity genes Tsc1 and Tsc2 were associated with susceptibility to Ptr ToxC- and Ptr ToxB-producing isolates, respectively, but Tsn1 was not associated with tan spot caused by Ptr ToxA-producing isolates, which further validates that the Tsn1−Ptr ToxA interaction does not play a significant role in tan spot development in durum. A unique locus on chromosome arm 2AS was associated with tan spot caused by race 4, a race once considered avirulent. A novel trait characterized by expanding chlorosis leading to increased disease severity caused by the Ptr ToxB-producing race 5 isolate DW5 was identified, and this trait was governed by a locus on chromosome 5B. We recommend that durum breeders select resistance alleles at the Tsr7, Tsc1, Tsc2, and the chromosome 2AS loci to obtain broad resistance to tan spot.",
    citingPapers: []
  },
  {
    title: "Genome-wide association mapping of resistance to the foliar diseases septoria nodorum blotch and tan spot in a global winter wheat collection.",
    authors: "Haugrud A, Shi G, Seneviratne S, Running K, Zhang Z, <strong>Singh G</strong>, Szabo-Hever A, Acharya K, Friesen TL, Liu Z, and Faris JD",
    journal: "<i>Molecular Breeding</i> 43:54",
    year: 2023,
    citations: 4,
    url: "https://link.springer.com/article/10.1007/s11032-023-01400-5",
    abstract: "<strong>Abstract:</strong> Septoria nodorum blotch (SNB) and tan spot, caused by the necrotrophic fungal pathogens Parastagonospora nodorum and Pyrenophora tritici-repentis, respectively, often occur together as a leaf spotting disease complex on wheat (Triticum aestivum L.). Both pathogens produce necrotrophic effectors (NEs) that contribute to the development of disease. Here, genome-wide association analysis of a diverse panel of 264 winter wheat lines revealed novel loci on chromosomes 5A and 5B associated with sensitivity to the NEs SnTox3 and SnTox5 in addition to the known sensitivity genes for NEs Ptr/SnToxA, SnTox1, SnTox3, and SnTox5. Sensitivity loci for SnTox267 and Ptr ToxB were not detected. Evaluation of the panel with five P. nodorum isolates for SNB development indicated the Snn3-SnTox3 and Tsn1-SnToxA interactions played significant roles in disease development along with additional QTL on chromosomes 2A and 2D, which may correspond to the Snn7-SnTox267 interaction. For tan spot, the Tsc1-Ptr ToxC interaction was associated with disease caused by two isolates, and a novel QTL on chromosome 7D was associated with a third isolate. The Tsn1-ToxA interaction was associated with SNB but not tan spot. Therefore some, but not all, of the previously characterized host gene-NE interactions in these pathosystems play significant roles in disease development in winter wheat. Based on these results, breeders should prioritize the selection of resistance alleles at the Tsc1, Tsn1, Snn3, and Snn7 loci as well as the 2A and 7D QTL to obtain good levels of resistance to SNB and tan spot in winter wheat.",
    citingPapers: []
  },
  {
    title: "Resistance screening and QTL mapping in wheat and triticale against root-lesion nematode.",
    authors: "<strong>Singh G</strong>",
    journal: "<i>MS Thesis</i>, North Dakota State University, Fargo, ND, USA",
    year: 2020,
    citations: 5,
    url: "https://www.proquest.com/openview/f06d4867c07d7e3ea21b2e2a9b5b6cb6/1?pq-origsite=gscholar&cbl=18750&diss=y",
    abstract: "<strong>Abstract:</strong> Root-lesion nematode (RLN, Pratylenchus neglectus) invades the roots of wheat and causes yield losses throughout the world. Genetic resistance is the most economical and effective means to manage RLNs. The objective of this study were to identify source of resistance to RLN in a small collection of wheat germplasm and to map quantitative trait loci (QTL) associated with RLN resistance in two; one wheat and one triticale recombinant inbred line (RIL) populations. Out of wheat lines, three were resistant, including hard red spring wheat cultivars Brennan, SY Ingmar, and SY Soren. A number of genomic regions in wheat and rye were identified as QTL for RLN resistance. My research provides a better understanding of the genetic basis of P. neglectus resistance and important tools for RLN resistance breeding.",
    citingPapers: []
  }
];
